MatrixSDK/MXRestClient.m

```diff
@@ -2694,6 +2694,35 @@ - (MXHTTPOperation*)relatedGroupsOfRoom:(NSString *)roomId
                            } failure:failure];
 }
 
+- (MXHTTPOperation*)roomSummaryWith:(NSString*)roomIdOrAlias
+                                via:(NSArray<NSString *>*)via
+                            success:(void (^)(MXPublicRoom *room))success
+                            failure:(void (^)(NSError *error))failure
+{
+    NSMutableString *path = [NSMutableString stringWithFormat:@"%@/im.nheko.summary/rooms/%@/summary", @"_matrix/client/unstable", roomIdOrAlias];
```

Comment:
Can we use constants for temporary path/types you are using here. We have `kMXAPIPrefixPathUnstable` for `@"_matrix/client/unstable"`